cmd/kube-proxy/app/server.go

```diff
@@ -205,15 +211,22 @@ func (o *Options) Validate(args []string) error {
 	if len(args) != 0 {
 		return errors.New("no arguments are supported")
 	}
+	return nil
+}
+
+func (o *Options) Run() error {
+	if c, err := o.loadConfigFromFile(o.ConfigFile); err != nil {
```

Comment:
please don't mutate Options in `Run()`... goal is to do mutation in `Complete()` so that inputs are resolved prior to calling `Run()`